src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -261,4 +241,21 @@ void parseCvrFile(String filePath, List<CastVoteRecord> castVoteRecords) {
       castVoteRecords.clear();
     }
   }
+
+  // Simple container class for contest data
+  public static class Contest {
```

Comment:
I believe you can (and should) make the class private, all the variables private, and even the constructor private (a grand total of six privates). 🚀 🎸 🍌 🍆 🥒 🔧

Reply:
Sure.  I suspect eventually this class will move out of here, but it can be private until then.